### PR TITLE
react-bookstore - Add style to the project

### PR DESCRIPTION
**Pull request by micheaol**
### In this pull request:

- I add style to the project

### CODE REVIEW

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hello 👋🏽  @micheaol 

Good job working on the project so far! 
However, there are some issues that you need to work on to go to the next project but you are almost there!

## Required Changes ♻️
While you've done a great job on the project, your design should be EXACTLY the same as the [Zeplin design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) so please make the following changes

- [ ] Add the person icon into the circle on the navbar.
- [ ] I should be able to see the add new book form on the main page and not when I click on categories. Kindly move the form away from categories and to the main page.
- [ ] Your app is missing the author names below the book title, please add it.
- [ ] There should be a vertical line separating comments, remove and edit from each other. Please add the line.
- [ ] Categories should have a colour of ``` #121212 ``` and not black, kindly change the colour.
- [ ] Also update progress text should be white and not black.  It should also be capitalized. Please refer to the [zeplin](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) design once more for the accurate colours and fonts.

### Your project
Screenshot 2021-12-10 at 10 05 03

### Zeplin design
Screenshot 2021-12-10 at 10 05 42

- [ ] It is more professional to have your styling in a file of its own and not in your components folder. Kindly create a CSS file/folder and move your styling there.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/BookCard/index.js [hunk 1]
```diff
@@ -20,6 +20,8 @@ import {
   BookLesson,
   UpdateProgress,
   BookLineDivider,
+  BookAuthor,
+  BookLineDividerSmall,
 } from './BookCardElements';
 
 const BookCard = ({ book, removeBookFromStore }) => (
```
Comment (SajjadAhmad14) [inline src/components/BookCard/index.js:27]: Styling should be exactly the same as in that Zeplin. Please make sure every style is the same.
Your progress bar:
progress
Zeplin:
zeplin

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Michael, 
 ### Thank you for working on changes suggested :ok_hand:
## To Improve 🙏
- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/components/BookCard/index.js (hunk 1 above)
Comment (micheaol) [reply src/components/BookCard/index.js:27]: Hi @SajjadAhmad14,

Thank you for the feedback. This issue was not raised by the last reviewer?

**Review by DeliceLydia — APPROVED** [review]
# Status: Approved🎉🎉

## Hi @micheaol, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Keep Rocking!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**